test/parallel/test-dgram-exclusive-implicit-bind.js

```diff
@@ -96,4 +99,12 @@ if (process.env.BOUND === 'y') {
   source.unref();
 }
 
-source.send(Buffer.from('abc'), 0, 3, common.PORT, '127.0.0.1');
+function send() {
```

Comment:
Could all of this just be simplified to:

```js
setInterval(() => {
  source.send(Buffer.from('abc'), 0, 3, common.PORT, '127.0.0.1');
}, 1).unref();
```

I think you shouldn't have to worry about clearing the interval if you unref it. That would reduce some complexity.